### PR TITLE
Change - to _ for context to make it consistent in the assembly

### PR DESCRIPTION
**Pull request by vikram-redhat**

### CODE REVIEW

File: modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ ifdef::context[:parent-context: {context}]
 
 // The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 [id='a-collection-of-modules']
-// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id='a-collection-of-modules-{context}'].
+// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id='a-collection-of-modules_{context}'].
```
Comment (ncbaratta) [inline modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc]: What are we making this consistent with? And why are we mixing hyphens and underscores?

**Comment by ncbaratta** [comment]
See the comments on PR #66

File: modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc (hunk 1 above)
Comment (vikram-redhat) [reply modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc]: @ncbaratta all other templates use _ (underscore) for connecting the id to the {context}. I assume this was a typo when they created this template and wanted to make sure we ware consistent. If the followup from #66 is to use - (dash) then let's keep this as a - dash as well.

**Comment by ritz303** [comment]
See PR #66 for more details.  Changed "_" --> "-".
https://github.com/redhat-documentation/modular-docs/commit/18bef9b2bd89279d562cefae4cebfc02fb19c215

**Pull request closed without merging**